README.md

```diff
@@ -20,7 +20,7 @@ I recommend switching over to ratatui since tui-rs is currently unmaintained.
 ```rust
 use ansi_to_tui::IntoText;
 let buffer = std::fs::read("ascii/text.ascii").unwrap();
-let output = buffer.into_text();
+let output = buffer.into_text().unwrap();
```

Comment:
Nit: Use ? instead of unwrap in docs as it promotes doing the right thing for newer rust users.

Reply:
Do you want me to change the line above too?

Reply:
Yeah sure. Perhaps also check lib.rs for the same problem though? If you don't have the inclination, feel free to leave it as unwrap and I'll clean it up at some point.

Reply:
easiest way in the lib.rs docs was a new dep but lmk if that's not acceptable